src/sas/sasgui/perspectives/fitting/fitting.py

```diff
@@ -1788,8 +1794,13 @@ def _complete1D(self, x, y, page_id, elapsed, index, model,
                     self._plot_residuals(page_id=page_id, data=data, fid=fid,
                                          index=index, weight=weight)
 
-            msg = "Computation  completed!"
+            msg = "Computation completed!"
+            if number_finite != y.size:
+                msg += ' PROBLEM: For some Q values the model returns non finite intensities!'
+                logger.error("For some Q values the model returns non finite intensities.")
+            
             wx.PostEvent(self.parent, StatusEvent(status=msg, type="stop"))
+            
         except:
```

Comment:
No need for try/accept block in

    try:
        ... code ...
    except:
        raise